boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -399,4 +409,147 @@ TEST_F(
     }
 }
 
+// DCM-8
+TEST_F(
+    DcmStateMachineTest,
+    regen_allowed_only_when_going_faster_than_5kph_and_airs_closed)
+{
+    SetInitialState(App_GetDriveState());
+
+    // Turn the DIM start switch on to prevent state transitions in
+    // the drive state.
+    EXPECT_TRUE(App_BuzzerSignals_IsOn(world));
```

Comment:
Checking the buzzer state is not related to this particular test so I'd remove this line too
```suggestion
```